base/shell/explorer/syspager.cpp

```diff
@@ -191,10 +192,27 @@ class CHardErrorThread
     CHardErrorThread():
         m_bThreadRunning(FALSE),
         m_Status(NULL),
```

Comment:
Would be m_Status(STATUS_SUCCESS)

Reply:
Why? This value would be rewritten one way or another before it gets used.